pkg/asset/cluster/azure/azure.go

```diff
@@ -0,0 +1,14 @@
+// Package azure extracts AZURE metadata from install configurations.
+package azure
+
+import (
+	"github.com/openshift/installer/pkg/types"
+	"github.com/openshift/installer/pkg/types/azure"
+)
+
+// Metadata converts an install configuration to Azure metadata.
+func Metadata(clusterID, infraID string, config *types.InstallConfig) *azure.Metadata {
```

Comment:
this is not using clusterID or infraID can we drop those from this function's api?

Reply:
of course, we can drop them for now.